site/jekyll/getting-started/tutorial_aspnet4.md

```diff
@@ -643,41 +654,45 @@ var CommentForm = React.createClass({
       </form>
     );
   }
-});
+}
 ```
 
 #### Events
 
 React attaches event handlers to components using a camelCase naming convention. We attach `onChange` handlers to the two `<input>` elements. Now, as the user enters text into the `<input>` fields, the attached `onChange` callbacks are fired and the `state` of the component is modified. Subsequently, the rendered value of the `input` element will be updated to reflect the current component `state`.
 
-(The astute reader may be surprised that these event handlers work as described, given that the method references are not explicitly bound to `this`. `React.createClass(...)` [automatically binds](/react/docs/interactivity-and-dynamic-uis.html#under-the-hood-autobinding-and-event-delegation) each method to its component instance, obviating the need for explicit binding.)
+You'll notice that we are explicitly binding our event handlers to `this` in the constructor. While older techniques, like `React.createClass(...)`, featured automatic binding, we are using ES6 classes to define our components. React components declared as ES6 classes don't automatically bind `this` to the instance, so we have to explicitly use `.bind(this)`.
```

Comment:
You can also use class field syntax instead of binding methods in the constructor. Let's add a link to this page which describes alternate patterns further:

https://reactjs.org/docs/handling-events.html

Reply:
I think this would still be good to include. I might put out another PR discussing this alternative. I didn't get it into this one before you merged it, but I can see this eventually being the preferred syntax once it stops being experimental.